packages/features/bookings/lib/getAllCredentialsForUsersOnEvent/getAllCredentials.ts

```diff
@@ -14,13 +13,13 @@ type User = Prisma.UserGetPayload<typeof userSelect>;
  *
  */
 export const getAllCredentials = async (
-  user: User & { credentials: CredentialPayload[] },
-  eventType: Awaited<ReturnType<typeof getEventTypesFromDB>>
+  user: { id: number; username: string | null; credentials: CredentialPayload[] },
+  eventType: { team: { id: number | null } | null; parentId: number | null } | null
```

Comment:
@zomars made some changes here. I think the types casted too broad of an umbrella. I narrowed it down to what's really needed. Thoughts?